pywinauto/unittests/test_Calendar.py

```diff
@@ -52,6 +49,10 @@ def test_can_get_current_date_from_calendar(self):
         date = self.calendar.get_current_date()
         self.assertThatSystemTimeIsEqualCurrentDateTime(date,datetime.date.today())
 
+    def test_should_throw_runtime_error_when_try_to_get_current_date_from_calendar_if_calendar_state_is_multiselect(self):
```

Comment:
Simple doc strings are also required for test methods (see Landscape warnings details in `Checks` section of the pull request).